README.md

```diff
@@ -0,0 +1,57 @@
+# ScyllaDB Web Install
+
+Welcome to ScyllaDB Web Install repository!  
+
+ScyllaDB Web Install is a platform-agnostic installation script that allows in easy way to install [ScyllaDB](https://www.scylladb.com/) product on different Linux OSs and on x86_64 / ARM64 machine types.  
+For more information, ScyllaDB Web Install user doc can be found [here](https://opensource.docs.scylladb.com/branch-5.4/getting-started/installation-common/scylla-web-installer.html).
+
+## Usage
+To install the latest official Open Source version of ScyllaDB by Web Installer script, run:
+```shell
+./server
+```
+The script will automatically detect your operating system and will install all the required packages / dependencies.  
+At the end of the successful installation, the message `Scylla installation done!` will be displayed.
+
+#### Additional options:
+- `[--scylla-version <version>]` - to install a specific ScyllaDB version.
+In `<version>`, specify `(x.y)` version to install the latest patch for this version or `(x.y.z)` to install a specific patch release. In addition, there is an option to install the latest nightly version by specify `nightly-<version>`.
+- `[--scylla-product <scylla|scylla-enterprise>]` - to install specific ScyllaDB product, the default is scylla OSS.
+- `[--scylla-repo-file-url <url>]` - allow to download and install ScyllaDB product from a custom `scylla.list` or `scylla.repo` url.
+- `[--verbose]` - to get more information during the installation run.
+- `[--dry-run]` - prints only the commands and the installation flow without actually installing it, useful for verification.
+
+#### Examples:
+Installing ScyllaDB Open Source version 5.2.10:
+```shell
+./server --scylla-product scylla --scylla-version 5.2.10
+```
+Installing ScyllaDB Enterprise latest patch of version 2023.1:
+```shell
+./server --scylla-product scylla-enterprise --scylla-version 2023.1
+```
+Running an installation of nightly version 4.5 in dry-run:
+```shell
+./server --scylla-version nightly-4.5 --dry-run
+```
+
+## Supported OSs and OS versions
+The following Operating Systems and OS versions are supported by this installer:
+
+| Linux Distributions             | OS Versions                   | 
+|---------------------------------|-------------------------------|
+| CentOS                          | 7.x                           |
+| CentOS Stream                   | 8.x , 9.x                     |
+| Oracle Linux                    | 7.x , 8.x                     |
+| Amazon Linux                    | 2                             |
+| Red Hat Enterprise Linux (RHEL) | 7.x , 8.x                     |
+| Rocky Linux                     | 8.x , 9.x                     |
+| Fedora Linux                    | 36 , 37, 38                   |
+| Debian Server                   | 9 , 10 , 11                   |
```

Comment:
We dropped the support for Debian 9, you can also remove it here

Reply:
The same here, it is supported by the script:
https://github.com/scylladb/scylladb-web-install/blob/956e7996bc6d857889e2e6e70b0485970eb484d5/server#L461-L467